packages/x-components/static-docs/build-search-ui/README.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Build your search UI for web
+tags:
+  - getting started
+  - x integration
+  - use x components
+  - x components
+  - interface x
+---
+
+::: slot left-intro-column
+
+## Getting started with Interface X Components
+
+Start building your search and discovery experience in a few simple steps:
+
+:::
+
+::: slot right-intro-column
+
+<img :src="$withBase('/assets/media/build-search-ui.svg')" alt="Build Your Search UI">
```

Comment:
I understand these images are supposed to be on the eDocs repo as I don´t see them here...but does that make sense?

Reply:
We are coupling the docs to the Docs Team technology. The same happened with Docusaurus. We could include the images in our repo, but that complicates the integration and the github actions. At the moment it only picks the .md files